cli/command/stack/deploy.go

```diff
@@ -164,6 +173,80 @@ func convertNetworks(
 	return nets
 }
 
+func convertVolumes(
+	serviceVolumes []string,
+	stackVolumes map[string]composetypes.VolumeConfig,
+	namespace string,
+) ([]mount.Mount, error) {
+	var mounts []mount.Mount
+
+	for _, volumeString := range serviceVolumes {
+		var (
+			source, target string
+			mountType      mount.Type
+			readOnly       bool
+			volumeOptions  *mount.VolumeOptions
+		)
+
+		// TODO: split Windows path mappings properly
+		parts := strings.SplitN(volumeString, ":", 3)
+
+		if len(parts) == 3 {
+			source = parts[0]
+			target = parts[1]
+			if parts[2] == "ro" {
+				readOnly = true
+			}
+		} else if len(parts) == 2 {
+			source = parts[0]
+			target = parts[1]
+		} else if len(parts) == 1 {
+			target = parts[0]
+		}
+
+		// TODO: catch Windows paths here
+		if strings.HasPrefix(source, "/") {
```

Comment:
I'm not incredibly comfortable with this ambiguity, but at least we'll get an error if the service lands on a node where the requested bind path doesn't exist.